pkg/cmd/service/service.go

```diff
@@ -74,13 +109,60 @@ type Config struct {
 	CORSAllowedHeaders []string `default:"*" split_words:"true"`
 }
 
-func GetServiceConfig() Config {
-	var config Config
+// GetServiceConfig returns the OpenFGA server configuration based on the values provided in the server's 'config.yaml' file.
+// The 'config.yaml' file is loaded from '/etc/openfga', '$HOME/.openfga', or the current working directory. If no configuration
+// file is present, the default values are returned.
+func GetServiceConfig() (Config, error) {
+
+	config := Config{
```

Comment:
Can we make a `func DefaultConfig() Config`  that returns this? I think that would be useful, especially for tests.

Reply:
👍 done!

Reply:
Ah yeah!